transports/msmq/index.md

```diff
@@ -71,12 +71,14 @@ Although MSMQ has the concept of both [Public and Private queues](https://techne
 
 ## Permissions
 
-| Group          | Permissions          |
-|----------------|----------------------|
-| Owning account | Write, Receive, Peek |
-| Administrators | Full                 |
-| Anonymous      | Write                |
-| Everyone       | Write                |
+| Group          | Permissions         | Description                       |
+|----------------|---------------------|-----------------------------------|
+| Owning account | Send, Receive, Peek | Set by NServiceBus                |
+| Administrators | Full                | Set by NServiceBus                |
+| Anonymous      | Send                | Set by NServiceBus prior to 6.1.0 |
+| Everyone       | Send                | Set by NServiceBus prior to 6.1.0 |
+
+NOTE: If you are running in *Workgroup* mode then Windows will grant Send permissions to Anonymous. Everyone is added by default by Windows but not with Send or Receive permisssions. Prior to NServiceBus 6.1.0 Send permissions for Anonymous and Everyone were granted at queue creation but this behavior was altered to not accidentally open up permissions.
```

Comment:
Minor change to my suggestion based on @mikeminutillo's feedback. 

NOTE: When running in *Workgroup* mode, Windows will grant `Send` permissions to the `Anonymous` and `Everyone` user group.  However, the `Receive` permissions are not explicitly granted. When running in *Domain* mode, Windows will *not* grant `Send` permissions to the `Everyone` user group. When running in this mode, the receiving endpoint's queue will need to explicitly grant `Send` permissions to the domain user accounts that is attempting to send a message.  

WARNING: When running Versions 6.0.10 and below,  `Send` permissions for `Anonymous` and `Everyone` group are granted at queue creation time.  While this might be ok in a development environment, the queue level permissions will need to be hardened in Production for better security.